ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -858,6 +865,69 @@ private ReplicaId getBootstrapReplicaInFullAuto(String partitionIdStr, DataNodeI
     }
   }
 
+  /**
+   * Return resource config from the cache if it's present, otherwise, fetch it from Helix.
+   * @param resourceName The resource name
+   * @param dcName The datacenter name
+   * @return The {@link ResourceConfig} object. If there is no config for the given resource name, null would be returned.
+   */
+  private ResourceConfig getResourceConfig(String resourceName, String dcName) {
+    if (!resourceConfigs.contains(resourceName)) {
+      ConfigAccessor configAccessor;
+      if (clusterMapConfig.clusterMapUseAggregatedView) {
+        configAccessor = helixAggregatedViewClusterInfo.helixManager.getConfigAccessor();
```

Comment:
Sorry, I missed this in older PRs. I think aggregated VIEW cluster only aggregates instance configs. Due to that, the resource configs would be empty. 

Since these are needed only on servers, may be we could use helixManager of local cluster always by exposing below as member variable?

`      localManager = initializeHelixManagerAndPropertyStoreInLocalDC(dataCenterToZkAddress, instanceName, helixFactory);`